### PR TITLE
Documenting LSTM options

### PR DESCRIPTION
**Pull request by jdiaz4302**
Quick demo that GPU is an option that has nearly identical training.

Likewise, the default `torch.nn.LSTM` is too and it is much faster, approximately 10x faster for the demonstrated hyperparameters while having very, very similar training.

Moving forward, I'm going to use `torch.nn.LSTM` rather than the other implementation (whose primary benefit was mimicking tensorflow's recurrent dropout functionality for people wanting a `torch` version of `river-dl`)

### CODE REVIEW

**Comment by jdiaz4302** [comment]
Automatically closing this PR, I'm only using it for documentation/later reference
